bin/run-integration-tests.sh

```diff
@@ -20,9 +20,7 @@ CMD="${CMD} -r a"
 CMD="${CMD} --verbose"
 CMD="${CMD} -n ${PYTEST_PROCESSES}"
 CMD="${CMD} --base-url ${BASE_URL}"
-# rerun a flaky test once
-# DO NOT INCREASE THIS
-CMD="${CMD} --reruns 1"
+CMD="${CMD} --reruns 2"
```

Comment:
This may explain why it's not working, according to the README for the pytest-rerunfailures package the number represents "the maximum number of times you'd like the test to run". So I think it was essentially off :(

https://github.com/pytest-dev/pytest-rerunfailures#re-run-all-failures